src/transformers/models/distilbert/modeling_distilbert.py

```diff
@@ -145,7 +154,19 @@ def prune_heads(self, heads):
         self.dim = attention_head_size * self.n_heads
         self.pruned_heads = self.pruned_heads.union(heads)
 
-    def forward(self, query, key, value, mask, head_mask=None, output_attentions=False):
+    def forward(
+        self,
+        query,
+        key,
+        value,
```

Comment:
It would be nice if we could refactor this method to just pass `hidden_states` instead of `query, key, value` similar to `BertSelfAttention`. 

For self attention `hidden_states` are used as query, key, and value and for cross-attention encoder_hiddem_states` are used as key and value.